src/utils/nodeTemplates.ts

```diff
@@ -138,6 +138,13 @@ export function createEmptyTuple(ast: AST): TupleExpression {
   return node;
 }
 
+export function createTuple(ast: AST, nodes: Expression[]): TupleExpression {
+  const typeString = `tuple(${nodes.map((node) => node.typeString)})`;
+  const node = new TupleExpression(ast.reserveId(), '', typeString, false, nodes, nodes);
```

Comment:
Why are you passing `nodes` as the value for the `raw` parameter? I'm not sure about what raw stores. Usually when creating a new node and there is no other node from where to extract its raw value, this is just left as undefined.